docs/syntax-reference.rst

```diff
@@ -1120,6 +1120,14 @@ Aggregate functions
 +------------------------+---------------------------+---------------------------------------------------------------------+
 | Function               | Example                   | Description                                                         |
 +========================+===========================+=====================================================================+
+| COLLECT_LIST           | ``COLLECT_LIST(col1)``    | Return an array containing all the values of ``col1`` from each     |
+|                        |                           | input row (for the specified grouping and time window, if any).     |
+|                        |                           | Currently only works for simple types (not Map, Array, or Struct).  |
++------------------------+---------------------------+---------------------------------------------------------------------+
+| COLLECT_SET            | ``COLLECT_SET(col1)``     | Return an array containing the distinct values of ``col1`` from     |
```

Comment:
Might be worth documenting it only works for streams, not tables...

Reply:
fair point. do you know of a place where we currently call out (or, better yet, actually explain) that distinction today, so i can add to it ? I couldn't find it on a cursory scan of the online docs

Reply:
It should be called out in the function blurb, IMHO.   If we put it else where its even more unlikely someone will read it.

Personally, I'm up for breaking this table into different sections / pages per function. Thereby allowing for more detailed info on each function.

I'd also call it out in the description you've got in the annotations on the classes themselves.

Reply:
i added this info to the per-udf annotation comments, as suggested, and also added a new column in the table which lists all aggregate functions in the syntax guide to indicate whether each one works on stream, tables, or both.